amity_app/classes/living_space.py

```diff
@@ -6,21 +6,28 @@ class LivingSpace(Room):
     """
     child class of class Room
     defines a room that is a living space
+    Contains create_room() the creates a room of type
+    living space
     """
 
     def __init__(self, room_type="livingspace", room_capacity=4):
-
```

Comment:
To really implement OOP, the properties of room that are common to both Living spaces and Offices can be defined in the Room class such as capacity, name and type.